backend/src/lib/server-side-events.ts

```diff
@@ -341,6 +341,8 @@ export class ServerSideEvents {
           break
         case 'Search':
         case 'Secret':
+        // // TODO: is other the proper case for AccessControl?
+        case 'AccessControl':
```

Comment:
This should be an actual kind name, like `ClusterPermission`. @jeswanke do you have thoughts on which group of events is the right one for ClusterPermission resources?

Reply:
is `other.push` the right one to push to? :thinking:

Reply:
`npm run check:fix` already executed and commit pushed

Reply:
`ClusterPermission` is actually the right one. Now the only concern I would say is about the `ServerSideEvent` type to push to (`other` ? )